src/Shared/KnownConfigNames.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting;
+
+internal static class KnownConfigNames
```

Comment:
KnownHostConfigNames

Reply:
Some of these values are going to likely be shared between components (e.g. dashboard and apphost). They represent the protocol which is why I wasn't getting specific to which process nominally owns them.